base/src/test/java/io/spine/protobuf/TypeConverterTest.java

```diff
@@ -56,123 +58,123 @@ protected void configure(NullPointerTester tester) {
         tester.setDefault(Any.class, Any.getDefaultInstance());
     }
 
-    @Test
-    @DisplayName("map arbitrary message to itself")
-    void map_arbitrary_message_to_itself() {
-        Message message = StringValue.of(newUuid());
-        checkMapping(message, message);
+    @Nested
+    @DisplayName("map")
+    class Map {
+
+        @Test
+        @DisplayName("arbitrary message to itself")
+        void map_arbitrary_message_to_itself() {
+            Message message = StringValue.of(newUuid());
+            checkMapping(message, message);
+        }
+
+        @Test
+        @DisplayName("Int32Value to int")
+        void map_Int32Value_to_int() {
+            int rawValue = 42;
+            Message value = Int32Value.of(rawValue);
+            checkMapping(rawValue, value);
+        }
+
+        @Test
+        @DisplayName("Int64Value to int")
+        void map_Int64Value_to_long() {
+            long rawValue = 42;
+            Message value = Int64Value.of(rawValue);
+            checkMapping(rawValue, value);
+        }
+
+        @Test
+        @DisplayName("FloatValue to float")
+        void map_FloatValue_to_float() {
+            float rawValue = 42.0f;
+            Message value = FloatValue.of(rawValue);
+            checkMapping(rawValue, value);
+        }
+
+        @Test
+        @DisplayName("DoubleValue to double")
+        void map_DoubleValue_to_double() {
```

Comment:
Please either fix the naming or create an issue.

Reply:
See #234.